spec/features/idv/steps/gpo_otp_verification_step_spec.rb

```diff
@@ -98,6 +98,64 @@
     end
   end
 
+  context 'warning alert banner for gpo letter spammed' do
```

Comment:
Since feature specs are rather slow and we've got three of them here, maybe it would be enough to have a view spec for this behavior instead? (i.e. in `spec/views/idv/by_mail/enter_code/index.html.erb_spec.rb`)